packages/devextreme/scss/widgets/base/treeView/_common.scss

```diff
@@ -1,21 +1,17 @@
 @use "../mixins" as *;
+@use "../../base/icons" as *;
 
 // adduse
 
-
 $treeview-searchbox-margin-bottom: 5px;
 $tree-view-icon-size: 24px;
 
 .dx-treeview-loadindicator-wrapper {
   text-align: center;
 }
 
-.dx-treeview-node-loadindicator {
-  position: absolute;
-}
-
 .dx-treeview-search {
-  margin-bottom: $treeview-searchbox-margin-bottom;
+  margin-block-end: $treeview-searchbox-margin-bottom;
```

Comment:
I think we shouldn't use margin-block-end here. Let's leave margin-bottom. Will this suitable for this case?

Reply:
yep